com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.hlsl

```diff
@@ -43,8 +44,14 @@ float4 FragBilinear(Varyings input) : SV_Target
     outColor = SAMPLE_TEXTURE2D(_SourceTex, sampler_LinearClamp, input.texcoord.xy);
 #endif
 
+    if (_SRGBRead && _SRGBWrite)
```

Comment:
iirc uint and bitfield op don't work on gles2 but I guess that might not be a problem for you. Just make sure this shader doesn't compile for gles2.

Reply:
I can confirm this shader does not compile for gles2: https://github.com/Unity-Technologies/Graphics/blob/12828751e058d5d82b944f387ea540d4ca02cdef/com.unity.render-pipelines.universal/Shaders/XR/XRMirrorView.shader#L8